samples/charts/pie-chart/legend/Pages/PieChartLegend.razor

```diff
@@ -0,0 +1,60 @@
+﻿@page "/PieChartLegend"
+
+@* this namespace is required for Infragistics controls *@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
```

Comment:
this DIV should have "igContainer-horizontal" class such that the legend is on left side and chart is on right side